README.md

```diff
@@ -261,6 +262,7 @@ this as the meter ID.
 - **SBC ALE3**: This compact Saia Burgess Controls meter is comparable to the SDM630.
 It has two tariffs, both import and export depending on meter version and compact (4TE). It's often used with Viessmann heat pumps.
 - **BE MPM3PM**: Compact (4TE) three phase meter.
+- **KOSTAL Smart Energy Meter**: Import Export data are currently not implemented.
```

Comment:
Gibts die bei dem Zähler/ sollten die hinzu gefügt werden?

Reply:
siehe auskommentierte Zeilen in `models.go` ich vermute einige von denen beinhalten die Im- bzw Export daten